File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        dic, res = {}, 0
        for i in s:
            if i not in dic:
                dic[i] = 1
            else:
                dic[i] += 1
        for i in t:
            if i not in dic:
                dic[i] = -1
            else:
                dic[i] -= 1
        for val in dic.values():
            if val < 0:
                res += (val * -1)
        return res

File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        map<int, int> ref;
        for (auto it : s) {
            ref[it]++;
        }
        for (auto it : t) {
            ref[it]--;
        }
        int res = 0;
        for (auto it : ref) {
            if (it.second < 0) res += (it.second * -1);
        }
        return res;
    }
};